### PR TITLE
src: drop HTTP from the gateway list

### PR DESCRIPTION
**Pull request by roman-khimov**
It's still in get_webstat_metrics.py, but upgrading this script is a bit more complicated, so let's just fix the UI.

### CODE REVIEW

File: src/Home.tsx [hunk 1]
```diff
@@ -236,10 +231,7 @@ const Home = ({
 												</thead>
 												<tbody>
 													{data.gateways && data.gateways[activeNet].map((node: string[]) => (
-														<tr key={node[0]}>
-															<td>
-																{node[0]}
-															</td>
+														<tr>
```
Comment (mike-petrov) [inline src/Home.tsx:234]: add `key={node[1]}`

Comment (roman-khimov) [reply src/Home.tsx:234]: I'm not sure why we have the `key` here. It doesn't make much sense to me.

Comment (mike-petrov) [reply src/Home.tsx:234]: Yes, it's done for react, otherwise it shows warning in the console

Comment (roman-khimov) [reply src/Home.tsx]: Huh. Surprising. Especially since I don't see it in the resulting HTML and it's just a table row. OK, fixed.

**Review by mike-petrov — APPROVED** [review]

**Pull request merged**